src/domain/child/__tests__/ChildUtils.test.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  getChildFormModalBirthdate,
+  getChildFormModalValues,
+} from '../ChildUtils';
+import { defaultChildFormBirthdate } from '../ChildConstants';
+import { defaultRegistrationData } from '../../registration/state/RegistrationReducers';
+
+describe('ChildUtil', () => {
+  describe('getChildFormModalBirthdate', () => {
+    test('should return child birthdate in format for form by default', () => {
+      const childForm = getChildFormModalBirthdate();
+
+      expect(childForm).toEqual(defaultChildFormBirthdate);
+    });
+
+    test('should return child birthdate in format for form by default when birthdate is empty string', () => {
+      const childForm = getChildFormModalBirthdate('');
+
+      expect(childForm).toEqual(defaultChildFormBirthdate);
+    });
+
+    test('should return child birthdate in format for form by default if given wrong format date', () => {
+      const childForm = getChildFormModalBirthdate('2020-02-31');
+
+      expect(childForm).toEqual(defaultChildFormBirthdate);
+    });
+
+    test('should give a nice form format birthdate data when given valid birthdate string', () => {
+      const childForm = getChildFormModalBirthdate('2020-02-27');
```

Comment:
Would be great to test leap year too! 2020-02-28 exists, but not 2021-02-28

Reply:
> ![image](https://user-images.githubusercontent.com/5602596/74159824-13541580-4c25-11ea-8baa-a5577ff1bc62.png)
> 
> I found a bug! The city is not prefilled. We should probably prefill it with Helsinki (en/fi)/Helsingfors(sv) if the variable is empty.

Added default homeCity for child edit form